venv/Include/Python.h

```diff
@@ -0,0 +1,159 @@
+#ifndef Py_PYTHON_H
```

Comment:
your python virtual environment should not be in git.  remove it from git and add this to the `.gitignore` file